llpc/context/llpcCompiler.cpp

```diff
@@ -975,6 +992,38 @@ bool Compiler::CanUseRelocatableGraphicsShaderElf(
     return useRelocatableShaderElf;
 }
 
+// =====================================================================================================================
+// Returns true if a compute pipeline can be built out of the given shader info.
+bool Compiler::CanUseRelocatableComputeShader(
```

Comment:
CanUseRelocatableComputeShader -> CanUseRelocatableComputeShaderElf
Keep it consistent with the naming of graphics shader